lib/js/importers/AllInOneImporter.ts

```diff
@@ -0,0 +1,189 @@
+const fsT = require('fs');
+const axios = require("axios");
+const tokensFilesConfig = JSON.parse(
+	fsT.readFileSync('lib/js/importers/configs/allInOneConfig.json', 'utf8')
+);
+
+interface IResultJsonObject {
+	id: string;
+	label: string;
+	value: string;
+	weight: number;
+}
+
+interface configFileObject {
+	destination: string,
+	destinationJson: string
+}
+
+interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+const ImportColorsRaw = (
+	binValues: configFileObject,
+	jsonSource: { data }
+) => {
+	let hexToCssVariable: Array<Object> = [];
+	try {
+		let result: Array<string> = [];
+		let resultJson = {};
+
+		jsonSource.data.colors.forEach(obj => {
+			const patternIsRawColor = /RAW\/|theme/i;
+			const patternDeprecated = /deprecated|Pattern/i;
```

Comment:
it isn't only deprecated. For me it looks like pattern of names to be ignored

Reply:
Updated.